src/inverseRobot.js

```diff
@@ -7,7 +7,18 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (robot[value]) {
+      return null;
+    }
+
+    robot[value] = key;
+    delete robot[key];
```

Comment:
instead of mutating the original object, it's better to create a new one, set proper keys and values and return it
![image](https://github.com/mate-academy/js_inverse_robot/assets/86923089/1b1c5666-51ad-4263-81d8-98f0d4d9149a)